developerportal/apps/articles/migrations/0032_auto_20190930_0915.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.4 on 2019-09-30 09:15
```

Comment:
Nitpick: don't worry about it here, but for future migrations, I find it's really useful to rename them to something explicit (eg `0032_update_article_manager.py`) - that makes them easier to grok when looking at `showmigrations`, etc

Reply:
Good point, it’s a bad habit I/we’ve fallen into. I’ll rename these before they land in master and will make sure to name them in the future.